environments/openshift-prod-cluster.sh

```diff
@@ -51,7 +51,8 @@ export FABRIC8_AUTH_API_URL=https://auth.openshift.io/api/
 export FABRIC8_FORGE_API_URL="https://forge.api.openshift.io"
 
 # this URL is used in dev mode only, for now feature-toggles.service.ts used WITH url as a proxy
```

Comment:
While we're touching this file:

"used WITH url as a proxy"

Maybe meant to be:

"uses WIT url as a proxy"

?